helper/acctest/random.go

```diff
@@ -134,47 +132,36 @@ func RandIpAddress(s string) (string, error) {
 		return prefix.Addr().String(), nil
 	}
 
-	prefixSizeExponent := uint(prefix.Addr().BitLen() - prefix.Bits())
-
-	if prefix.Addr().Is4() && prefixSizeExponent > 31 {
-		return "", fmt.Errorf("CIDR range is too large: %d", prefixSizeExponent)
+	// base address as byte slice
+	prefixBytes, err := prefix.Masked().Addr().MarshalBinary()
```

Comment:
I've added `Masked()` here to ensure that a user asking for a random address within "192.168.1.77/24" isn't masking out random bits with that "77". They'll get any address within 192.168.1.0-255 this way.

Reply:
Ah! That makes sense. I can't really think of a scenario where you'd _want_ to lose that portion of the range, and honestly at that point, the cost of implementing your own random IP function is low 😄